src/librustc/mir/visit.rs

```diff
@@ -71,7 +71,7 @@ macro_rules! make_mir_visitor {
             // Override these, and call `self.super_xxx` to revert back to the
             // default behavior.
 
-            fn visit_mir(&mut self, mir: & $($mutability)? Mir<'tcx>) {
+            fn visit_mir(&mut self, mir: & $($mutability)? Body<'tcx>) {
```

Comment:
This and `super_mir` should be renamed to `visit_body`, and `super_body`, respectively.

Reply:
Rodger that. Done in the last force push.